### ISSUE TITLE
Versioning number change

### ISSUE THREAD

**Issue by lboynton** [OpsLevel/terraform-provider-opslevel#329]
Hi

I noticed the versioning convention has changed, for example the latest release is [v0.13.0-1](https://github.com/OpsLevel/terraform-provider-opslevel/releases/tag/v0.13.0-1) which has a `-1` suffix. Up to [v0.11.0](https://github.com/OpsLevel/terraform-provider-opslevel/releases/tag/v0.11.0) it was using a different convention.

The current convention doesn't seem to match the terraform convention shown [here](https://developer.hashicorp.com/terraform/language/expressions/version-constraints#version-constraint-syntax).

> Version numbers should be a series of numbers separated by periods (like 1.2.0), optionally with a suffix to indicate a beta release.

As a result dependency tools like Renovate are not detecting the new versions since `v0.11.0` without a [manual override](https://docs.renovatebot.com/modules/versioning/#examples-of-versioning-overrides). Is it possible to go back to the old versioning system?

**Comment by rocktavious** [comment]
Hey @lboynton - yep and this was intentional and expected.  We are currently undergoing a massive rewrite under the hood to upgrade the provider from terraform SDK V1 -> V3.  The `-1` added to the end is a "pre-release" according to SemVer standards (and we utilize a [version bumping action](https://github.com/zwaldowski/semver-release-action) to facilitate this) and its expected that tooling like Renovate is not going to recommend upgrading to a pre release.

Once we feel confident in the pre-release to move it to a real release we will be going 1.0 and some information will be coming out about that in 2 weeks.

PS. We did have a problem with the semver release action so we ended releasing `v0.12.0-0` and then next cycle `v0.13.0-0` so thats why there is no `v0.12.0` non prerelease.  Rest assured there are no major changes you are missing out on yet we are still rewriting on the new SDK trying to ensure as much backwards compatibility as possible but also fixing numerous long standing bugs that were do to SDK V1 that are not fixed in SDK V3
